src/ui/style/cssvars.css

```diff
@@ -16,6 +16,13 @@
   --rabby-light-orange-default: rgba(255, 176, 32, 1);
   --rabby-light-orange-light: rgba(255, 245, 226, 1);
   --rabby-light-orange-DBK: rgba(255,124,96, 1);
+  --rabby-light-sonic-btn: rgba(33, 78, 129, 0.16);
+  --rabby-light-sonic-btn-foreground: rgba(33, 78, 129, 1);
+  --rabby-light-sonic-card: rgba(255, 255, 255, 1);
+  --rabby-light-sonic-card-foreground: rgba(0, 0, 0, 1);
+  --rabby-light-sonic-card-border: rgba(80, 97, 121, 0.1);
+  --rabby-light-sonic-background: rgba(245, 245, 245, 1);
+  --rabby-light-sonic-foreground: rgba(0, 0, 0, 1);
```

Comment:
same as `theme-colors.js`, please move to the bottom

Reply:
Says the file is autogenerated?